doc/source/categorical.rst

```diff
@@ -370,7 +370,7 @@ Renaming categories is done by assigning new values to the
 
 .. note::
 
-    Be aware that assigning new categories is an inplace operation, while most other operations
+    Be aware that assigning new categories is an in place operation, while most other operations
```

Comment:
Still think inplace as a term is fine, especially since it aligns with the keyword used for the concept throughout pandas